bgpd/bgp_mplsvpn.c

```diff
@@ -3752,14 +3752,18 @@ vrf_id_t get_first_vrf_for_redirect_with_rt(struct ecommunity *eckey)
  * This function gets called when the default instance ("router bgp NNN")
  * is created.
  */
-void vpn_leak_postchange_all(void)
+static void vpn_leak_postchange_all_internal(struct event *t
+					     __attribute__((__unused__)))
 {
 	struct listnode *next;
 	struct bgp *bgp;
 	struct bgp *bgp_default = bgp_get_default();
 
 	assert(bgp_default);
 
+	EVENT_OFF(bm->t_vpn_leak_postchange);
```

Comment:
How is this even possible?  Also how is the next line needed?  Please remove these two lines.

Reply:
ok this function can be called with an event and without.  Leave the EVENT_OFF but remove the null set, event off does this.

Reply:
done